.github/scripts/quarkus-nightly.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Could this be handled by a maven profile that is activated based upon an environment variable?  The action.yml could use the ${{ github.event.inputs.is-quarkus-nightly == 'true'}} expression to define the variable.